### ISSUE TITLE
Latest version not available on the Forge

### ISSUE THREAD

**Issue by smortex** [xaque208/puppet-bacula#161]
Hi @xaque208 

It looks like the forge has version 5.7.0 while 5.8.0 was tagged last summer…  can we upload it to the forge manually?

Publishing seems to be the responsibility of Travis-CI which recently chose to change it's term of service.  With the @opus-codium folks, we switched to GitHub action and took that opportunity to rely on @voxpupuli's work with GitHub actions and releasing our module was actually smoother than expected.

If you are interested in such a move and would like to have a buddy to give a hand, feel free to ask :wink:

**Comment by xaque208** [comment]
Hi @smortex, I've built and pushed 5.8.0 to the forge, thank you for the nudge, and apologies for the delay. :)

Github actions for a release sound great.  I've been using them in my professional job for a while now, so I'm +1 to get them going for puppet modules too.

I've started again at keeping a fork of modulesync configs here: https://github.com/xaque208/modulesync_config/pull/4

I'll start running through the modules and work on getting the build passing.  I dread the updates because things tend to break in ways I don't expect, but I certainly appreciate the work of the @voxpupuli group, and I was glad to see an update for the github workflows.  I'd been rebasing the modulesync repo, but I think if I just merge that PR, I can pull in the changes from upstream without the conflicts.

Anyway, thanks for the patience for this perpetual release friction.  Any thoughts on how I/we could improve the process here I'm all ears as usual.  Perhaps the outline above is a step in the right direction for reducing the maintenance load for the these modules.
